UnitTests/BraintreePayPalTests/BTPayPalApprovalURLParser_Tests.swift

```diff
@@ -0,0 +1,48 @@
+import XCTest
+@testable import BraintreeCore
+@testable import BraintreePayPal
+
+final class BTPayPalApprovalURLParser_Tests: XCTestCase {
+
+    func testInit_withPayPalAppApprovalUrl_setsRedirectTypeAndBAToken() {
```

Comment:
* [Test for parsing baToken](https://github.com/braintree/braintree_ios/blob/f7ed71ea853aeee0cc77cf445068ab203aa864b9/UnitTests/BraintreePayPalTests/BTPayPalClient_Tests.swift#L276-L294)
* [Test for "redirect type" / app switch was attempted](https://github.com/braintree/braintree_ios/blob/f7ed71ea853aeee0cc77cf445068ab203aa864b9/UnitTests/BraintreePayPalTests/BTPayPalClient_Tests.swift#L729-L763)